Update mobile release notes

UrazAkgultan opened the pull request.

UrazAkgultan committed:
update mobile release notes

ConnorLand commented on content/en/docs/releasenotes/mobile/_index.md:
```diff
@@ -6,7 +6,8 @@ weight: 15
 This category includes the following release notes: 
 
-* Our [Make It Native Apps](/releasenotes/mobile/make-it-native-parent/) document links to both versions of the Make It Native app:
+* Our [Make It Native Apps](/releasenotes/mobile/make-it-native-parent/) document links to all versions of the Make It Native app:
```
Thanks so much for catching this @UrazAkgultan

ConnorLand committed:
Update nt-8-rn.md

ConnorLand committed:
Update make-it-native-10.md

ConnorLand committed:
higher and higher!

ConnorLand committed:
Update make-it-native-10.md

ConnorLand committed:
Remove KI until the KI process is done

The following text was removed:

Augmented Reality (AR) Feature: We are currently aware of an issue where the AR functionality is not operating as expected in the MiN app. Our team is actively working on a solution and we aim to fix this in an upcoming update.

ConnorLand committed:
Change MIN and Atlas Core links in 10.6 RN
